enhancements/platforms/ovirt-rhv-ipi.md

```diff
@@ -0,0 +1,271 @@
+  the installer - an IPI installation.
+- implementing a cluster-api provider to provide scaling and managing the cluster
+  nodes (used by IPI, and useful for UPI, and also node management/fencing)
+
+### Non-Goals
+- UPI implementation will be provided separately.
+
+## Proposal
+
+This provider enables the Openshift Installer to provision VM resources in an \
+oVirt data center, that will be used as worker and masters of the clusters. It \
+will also create the bootstrap machine, and the configuration needed to get \
+the initial cluster running by supplying DNS a service and load balancing, all \
+using static pods.
+
+This work is related to the Bare-Metal provider because oVirt does not supply \
+DNS and LB services but is a platform provider. See the [baremetal ipi networking doc][baremetal-ipi-networking]
+
+### Implementation Details/Notes/Constraints [optional]
+
+1. Survey
+
+The installation starts and right after the user supplies his public ssh key,\
+and then choose `ovirt` the installation will ask for all the relevant details\
+of the installation: **url**, **user** and **password** of ovirt api. The installer \
+will validate it can communicate with the api, otherwise it will fail to proceed.\
+Next the installer will ask to choose the ovirt **cluster**, where the VMs will be\
+created, and next would be a *template* from that cluster for those VMs.\
+*Note* that it is the user's responsibility to upload the relevant RHCOS template\
+to ovirt. This may change in the future.
+With that the survery continues to the general cluster name, domain name, and\
+the rest of the non-ovirt specific question.
+
+2. Resource creation - terraform
+
+All the information from the survey is serialized by the installer into the file\ `terraform.ovirt.auto.tfvars.json` under the current directory and then it invokes\
+terraform, which automatically load `.auto.tfvars.json`. `terraform.tfstate` is\
+the actual declaration of the resources and with it terraform creates the VMs.\
+
+3. Bootstrap
+
+The bootstrap VM has a huge ignition config set using terraform and is visible\
+in the `terraform.tfstate` file. oVirt boots that VM with that content and the\
+bootstraping begins when the `bootkube.service` systemd service starts.\
+The bootstrap job is to:
+ 1. run a DNS service which will listen on the DNS VIP supplied by the survey\
+    to allow internal resolution of the api and resolution of each node's FQDN\
+ 2. run keepalived to activate the DNS VIP and API VIP on the machine nic as\
+    secondary IP. 
+ 3. run and expose machine configs, using a static pod of Machine Config Server\
+    listening on the API VIP
+ 4. load and bootstrap etcd service and help sign etcd members on the masters\
+    and boot staticly the rest of the cluster pods.
+ 5. pivot to a real master after its done
+
+4. Masters bootstrap
+
+Master VMs are booting using a thin ignition config that are waiting early in\
```

Comment:
nit: `Ignition`, the `I` needs to be capital.

Reply:
done